hudi-common/src/test/java/org/apache/hudi/common/testutils/HoodieTestTable.java

```diff
@@ -286,8 +286,8 @@ public HoodieTestTable addClean(String instantTime, HoodieCleanerPlan cleanerPla
   }
 
   public HoodieTestTable addClean(String instantTime, HoodieCleanerPlan cleanerPlan, HoodieCleanMetadata metadata, boolean isEmpty) throws IOException {
-    createRequestedCleanFile(basePath, instantTime, cleanerPlan, isEmpty);
-    createInflightCleanFile(basePath, instantTime, cleanerPlan, isEmpty);
+    createRequestedCleanFile(basePath, instantTime, cleanerPlan, false);
+    createInflightCleanFile(basePath, instantTime, cleanerPlan, false);
```

Comment:
why changing this?

Reply:
empty requested and empty inflight should be addressed by action executors. only completed being empty can be encountered by archival. and thats why.